alicloud/data_source_alicloud_security_groups.go

```diff
@@ -10,11 +10,6 @@ import (
 	"github.com/hashicorp/terraform/helper/schema"
 )
 
-type SecurityGroup struct {
-	Attribute    ecs.DescribeSecurityGroupAttributeResponse
-	CreationTime util.ISO6801Time
-}
-
 func dataSourceAlicloudSecurityGroups() *schema.Resource {
```

Comment:
HI, I am sorry that I give you a incorrect review. I think security group CreationTime can be got from ecs.DescribeSecurityGroupAttributeResponse, and new type SecurityGroup is needless. But it is my fault and CreationTime does not exist in DescribeSecurityGroupAttributeResponse. So may be you should return. I am sorry again.

Reply:
Hi, no problem at all. Thank you for the review and for clarification. Please find my update.